src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	
+	Name name;
```

Comment:
the Tagging association should identify a person and a tag, assume the application allows different people with same name, by just using name here, you wouldn't know which person was tagged with the given tag, no? what is a better way?